tensorflow_transform/info_theory.py

```diff
@@ -14,7 +14,7 @@
 """Utilities for information-theoretic preprocessing algorithms."""
 
 import math
-
+import numpy as np
```

Comment:
Note that this is a library based on tensorflow, so all of the math operations done below are deferred by constructing the tensorflow graph, and should not be using numpy.
There are tensorflow alternatives for these operations though.

Were you able to run the tests in this repo to make sure that they don't break with the change?

Reply:
Thank you for your response. I'm new to this repository. Before adding the new import, I checked to see if this repo makes use of numpy APIs (https://github.com/tensorflow/transform/search?q=numpy). Therefore , I automatically analyzed the code to identify any missed chances to use the APIs. Please close the PR if this script is not intended to use numpy APIs. Regarding test. No I did not. Now I'm trying to run the test. However, I am having difficulty locating the test scripts. Any assistance is much appreciated.